test/e2e/utils/datainserter.go

```diff
@@ -113,6 +114,16 @@ func (di *DataInserter) Insert() error {
 	return nil
 }
 
+func (di *DataInserter) AwaitSchemaAgreement(ctx context.Context) error {
+	framework.Infof("Awaiting schema agreement")
+	if err := di.session.AwaitSchemaAgreement(ctx); err != nil {
```

Comment:
tiny nit: separate assignment for consistency

Reply:
done